tests/test_modeling_common.py

```diff
@@ -178,7 +178,6 @@ def _config_zero_init(config):
 
 @require_torch
 class ModelTesterMixin:
-
```

Comment:
This file should not be changed

Reply:
The changes happen when I format the code using `black.` If I don't push these changes, errors regarding code quality show up.

Reply:
Hmm that should normally not happen. Could you copy test_modeling_common.py from Github and run make style?

What's your black version?

Reply:
I am using `black==22.10.0`.

> Could you copy test_modeling_common.py from Github and run make style?
I also tried doing this. The `make style` command formats the file just like in this PR.

Reply:
`transformers` is using `black==22.3` (`pip install -e .[dev]` if you're in doubt).